sbi/inference/snl/snl.py

```diff
@@ -136,20 +143,24 @@ def __call__(self, num_rounds: int, num_simulations_per_round):
             # Generate parameters from prior in first round, and from most recent posterior
             # estimate in subsequent rounds.
             if round_ == 0:
-                parameters, observations = simulators.simulation_wrapper(
+                parameters, observations = simulators.simulation_wrapper_batch(
```

Comment:
I find it confusing that the `simulation_wrapper_batch` is called from `simulators`, I think this is because of the way it is imported. Shall we just import it explicitly as a function?